boards/wch/ch32v003evt/board.yml

```diff
@@ -0,0 +1,5 @@
+board:
+  name: wch_ch32v003evt
```

Comment:
```suggestion
  name: wch_ch32v003evt
  full_name: WCH CH32V003EVT
```

Reply:
The `full_name` field was added - thanks!